notebooks/vertex_genai/solutions/vertex_llm_evaluation.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        "id_columns": ["id", "document"],
</pre><p>I guess only id is sufficient if it is truly a unique id.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Agreeing. This is weird. I'll try with id only see if that works. Otherwise, I'll revert. </p>